src/geo/gmaps/gmaps.js

```diff
@@ -226,7 +226,11 @@ if(typeof(google) != "undefined" && typeof(google.maps) != "undefined") {
     },
 
     pixelToLatLon: function(pos) {
-      return this.projector.fromContainerPixelToLatLng(new google.maps.Point(pos[0], pos[1]));
+      var latLng = this.projector.pixelToLatLng(new google.maps.Point(pos[0], pos[1]));
+      return {
+        lat: latLng.k,
```

Comment:
I think k and D variables come from the google ofuscator, I'd use lat() and lng() methods:

https://developers.google.com/maps/documentation/javascript/reference?hl=es see "google.maps.LatLng class"